tests/unittests/test_interfaces.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+import torch
+from hyperpyyaml import load_hyperpyyaml
+
+
+def test_load_audio(tmpdir):
+    from speechbrain.pretrained.interfaces import Pretrained
+
+    hparams_file = (
+        Path("tests")
+        / "integration"
+        / "neural_networks"
+        / "ASR_seq2seq"
+        / "hyperparams.yaml"
+    )
+    with open(hparams_file) as fin:
+        hparams = load_hyperpyyaml(fin)
+
+    pretrained = Pretrained(hparams["modules"], hparams)
```

Comment:
Dropped my earlier comment here - all that is relevant to be tested is:
```python
        source, fl = split_path(path)
        path = fetch(fl, source=source, savedir=savedir)
        signal, sr = torchaudio.load(path, channels_first=False)
```

Loading a 100+ MB pretrained model is too much time in preparing this unit test for what needs to be done. (Suggested earlier to use a smaller pretrained model but that's also too much.)